applications/vanilla/views/discussions/table_functions.php

```diff
@@ -58,7 +58,8 @@ function writeDiscussionRow($Discussion, $Sender, $Session) {
         $Sender->fireEvent('BeforeDiscussionName');
 
         $DiscussionName = $Discussion->Name;
-        if ($DiscussionName == '') {
+        // If there are no word character detected in the title treat it as if it is blank.
+        if (preg_match('/\w/u', $DiscussionName) !== 1) {
```

Comment:
Is using an is-identical comparison that much faster?  I think pretty much everything in Vanilla that uses `preg_match` currently just tests for the result being truthy/falsey. If it's a real benefit, I think it's worth using, going forward.  As it stands, these comparisons stick out with similar conditionals in core and I want to make sure it's for a good reason.